Sources/Diffing.swift

```diff
@@ -40,3 +41,118 @@ public extension DiffableViewModel {
         return String(describing: Self.self)
     }
 }
+
+/// MARK: - DifferenceKit Helpers
+
+/// Creates a type-erased Differentiable for DiffableViewModel.
+/// These are only created internally from either `TableCellViewModel` or `CollectionCellViewModel`,
+/// so that we can safely force cast the models back to those types.
+public struct AnyDiffableViewModel {
```

Comment:
Can some of these helper types be made non-public?

Reply:
Nope. They're used in public methods/vars that are requirements of public protocols.

Reply:
If this struct is only used internally, can it be `internal` instead of `public`? 

Maybe I'm misreading your comment?

Reply:
`AnyDiffableViewModel` is used in a decl that satisfies a protocol for a public type, so this has to be public.

Reply:
Ah, got it 👍